### ISSUE TITLE
Python3.7 support

### ISSUE THREAD

**Issue by jgc128** [kmike/pymorphy2#110]
Hi,

Building with pip (`pip install pymorphy2[fast]`) fails, probably because of https://github.com/pytries/DAWG/issues/31

Installation with `pip install pymorphy2` works but behaves really weird - sometimes it gives `Segmentation fault`. I couldn't determine what precisely causes this error - it just fails after processing of some portion of the data

**Comment by opennota** [comment]
AFAIK pypi.org has an old unsupported version of pymorphy2 (`0.8`). Try installing pymorphy2 from github:

`pip install git+https://github.com/kmike/pymorphy2.git`

**Comment by jgc128** [comment]
Yes, I reported this issue just in case the author does not know about such problem with python3.7

**Comment by kiddick** [comment]
same here
